recipes/scikit-misc/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "scikit-misc" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/has2k1/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: a3be820eaecef4bedc8577741481e0e99c001ad0142df5096fceaebe52e44233
+
+build:
+  number: 0
+  skip: True  # [win or py27]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+  host:
+    - python
+    - pip
+    - cython
+    - numpy
+  run:
+    - python
+    - numpy
+    - libopenblas
```

Comment:
Is the explicit dependency on libopenblas really needed? Numpy will pull it in if *it* needs it.

Reply:
Without explicitly stating `libopenblas`, my local test on Linux give the following error:

```
Packaging scikit-misc
INFO conda_build.build:build(2214): Packaging scikit-misc
Packaging scikit-misc-0.1.3-py38h0e9072a_0
INFO conda_build.build:bundle_conda(1454): Packaging scikit-misc-0.1.3-py38h0e9072a_0
compiling .pyc files...
number of files: 32
WARNING :: get_rpaths_raw()=['/home/conda/staged-recipes/build_artifacts/scikit-misc_1608693257402/_build_env/lib:/home/conda/staged-recipes/build_artifacts/scikit-misc_1608693257402/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib'] and patchelf=[['/home/conda/staged-recipes/build_artifacts/scikit-misc_1608693257402/_build_env/lib', '/home/conda/staged-recipes/build_artifacts/scikit-misc_1608693257402/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib']] disagree for /home/conda/staged-recipes/build_artifacts/scikit-misc_1608693257402/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.8/site-packages/skmisc/loess/_loess.cpython-38-x86_64-linux-gnu.so :: 
Warning: rpath /home/conda/staged-recipes/build_artifacts/scikit-misc_1608693257402/_build_env/lib is outside prefix /home/conda/staged-recipes/build_artifacts/scikit-misc_1608693257402/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold (removing it)
   INFO: sysroot: '/home/conda/staged-recipes/build_artifacts/scikit-misc_1608693257402/_build_env/x86_64-conda-linux-gnu/sysroot/' files: '['/home/conda/staged-recipes/build_artifacts/scikit-misc_1608693257402/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/zone.tab', '/home/conda/staged-recipes/build_artifacts/scikit-misc_1608693257402/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/tzdata.zi', '/home/conda/staged-recipes/build_artifacts/scikit-misc_1608693257402/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/right/Zulu', '/home/conda/staged-recipes/build_artifacts/scikit-misc_1608693257402/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/right/WET']'
  ERROR (scikit-misc,lib/python3.8/site-packages/skmisc/loess/_loess.cpython-38-x86_64-linux-gnu.so): Needed DSO lib/libopenblas.so.0 found in ['libopenblas']
  ERROR (scikit-misc,lib/python3.8/site-packages/skmisc/loess/_loess.cpython-38-x86_64-linux-gnu.so): .. but ['libopenblas'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (scikit-misc,lib/python3.8/site-packages/skmisc/loess/_loess.cpython-38-x86_64-linux-gnu.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-h77966d4_13
   INFO (scikit-misc,lib/python3.8/site-packages/skmisc/loess/_loess.cpython-38-x86_64-linux-gnu.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-h77966d4_13
WARNING (scikit-misc): plugin library (Python) package conda-forge::numpy-1.16.6-py38h18fd61f_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (scikit-misc): run-exports library package conda-forge::libgfortran5-9.3.0-he4bcb1c_17 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (scikit-misc): interpreter (Python) package conda-forge::python-3.8.6-hffdb5ce_2_cpython in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
Traceback (most recent call last):
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 157, in <module>
    build_all(os.path.join(root_dir, "recipes"), args.arch)
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 81, in build_all
    build_folders(recipes_dir, new_comp_folders, arch, channel_urls)
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 138, in build_folders
    conda_build.api.build([recipe], config=get_config(arch, channel_urls))
  File "/opt/conda/lib/python3.8/site-packages/conda_build/api.py", line 186, in build
    return build_tree(
  File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 3008, in build_tree
    packages_from_this = build(metadata, stats,
  File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 2291, in build
    newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
  File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 1584, in bundle_conda
    files = post_process_files(metadata, initial_files)
  File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 1430, in post_process_files
    post_build(m, new_files, build_python=python)
  File "/opt/conda/lib/python3.8/site-packages/conda_build/post.py", line 1319, in post_build
    check_overlinking(m, files, host_prefix)
  File "/opt/conda/lib/python3.8/site-packages/conda_build/post.py", line 1225, in check_overlinking
    return check_overlinking_impl(m.get_value('package/name'),
  File "/opt/conda/lib/python3.8/site-packages/conda_build/post.py", line 1208, in check_overlinking_impl
    raise OverLinkingError(overlinking_errors)
conda_build.exceptions.OverLinkingError: overlinking check failed 
["  ERROR (scikit-misc,lib/python3.8/site-packages/skmisc/loess/_loess.cpython-38-x86_64-linux-gnu.so): .. but ['libopenblas'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)"]

```

Reply:
@dopplershift Do you have any suggestion regarding this issue?

Reply:
Sounds like adding `libopenblas` might just be the right answer then.